libtelemetry/src/main/java/com/mapbox/android/telemetry/RevokedKeys.java

```diff
@@ -0,0 +1,12 @@
+package com.mapbox.android.telemetry;
+
+import java.util.List;
+
+class RevokedKeys {
```

Comment:
What are some benefits and side-effects of having this class defined? It looks like it just simply wraps the list of keys.